schemes/chemistry/prescribed_ozone.meta

```diff
@@ -0,0 +1,162 @@
+[ccpp-table-properties]
+  name = prescribed_ozone
+  type = scheme
+  dependencies = ../../../../history/cam_history.F90,../../../../utils/tracer_data.F90
```

Comment:
As discussed in the RRTMGP PR, I don't think we need to list CAM-SIMA specific `dependencies` here (only those that are other schemes/utils in atmospheric_physics itself).

Reply:
Thanks, removed throughout!